modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/EventInput.js

```diff
@@ -116,6 +128,43 @@ function EventInput({
 					value={value}
 				/>
 
+				<ClaySelectWithOption
+					aria-label={`${propertyLabel}: ${Liferay.Language.get(
+						'select-date-operator-option'
+					)}`}
+					className="criterion-input form-control"
+					defaultValue={DATE_OPERATORS.EVER}
+					disabled={disabledInput}
+					onChange={(event) =>
+						onChange({
+							day: {
+								...criterion.day,
+								operatorName: event.target.value,
+							},
+						})
+					}
+					options={SUPPORTED_EVENT_DATE_OPERATORS.map(
+						({label, name: value}) => ({label, value})
+					)}
+					value={criterion.day.operatorName}
```

Comment:
I think that this value will not work with the ever modifier 🤔

Reply:
:thinking: I need to have a look. I think I saw it working but I am probably wrong. I'll check that :smile: